apache/wsgi-py3.conf.in

```diff
@@ -158,10 +158,11 @@ Header always set Access-Control-Allow-Headers "x-requested-with, Content-Type,
 # kml file downloads
 AliasMatch ^${APACHE_ENTRY_PATH}/kml/(map\.geo\.admin\.ch_KML_\d+\.kml)$ ${KML_TEMP_DIR}/$1
 
-# KML admin
+# KML admin TODO KML
 RewriteCond %{HTTP:X-Forwarded-Proto} !https
 RewriteRule ^${APACHE_ENTRY_PATH}/admin         https://%{HTTP_HOST}%{REQUEST_URI} [R,L]
 
+# TODO KML
 <LocationMatch ^${APACHE_ENTRY_PATH}/(admin/kml|static/info.json)>
     AuthType Basic
     AuthName "Your IWI credential"
```

Comment:
@procrastinatio @boecklic same here the LocationMatch can be removed ? What is the `static/info.json` ? I guess this needs to be kept ?

Reply:
`info.json` is a file with version info (build version, date, etc.)

Reply:
If you remove this line, remove everything concerning `info.json`